cmd/monaco/download/download_configs.go

```diff
@@ -238,16 +242,19 @@ func copyConfigs(dest, src project.ConfigsPerType) {
 	}
 }
 
-// shouldDownloadClassicConfigs returns true unless onlySettings or specificSchemas but no specificAPIs are defined
```

Comment:
I've removed the comments as it became a bit awkward to update them and write a new one for shouldDownlaodAutomationResources